portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,61 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Small Business </title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+      
+<ul>
+  <li><a class="active" href="#home">Home</a></li>
+  <li><a href="#news">News</a></li>
+  <li><a href="#contact">Contact</a></li>
+  <li><a href="#about">About</a></li>
+</ul>
+    <div class="pimg1">
+      <div class="ptext">
+        <span class="border trans">
+         Welcome !!!
+        </span>
+            
+            </div>
+      </div>
+        </div>
+    
+
+    <section class="section section-light">
+                
+                  <div class="ptext">
+                    <span class="border trans">
+                    Small Business Locater Form  
+                    </span>
+                  </div>
+                  <div class= "form">
+                      TBD
+
+                  </div>
+                  
+                
+
+      </section>
+
+
+                       <div class="pimg3"> 
```

Comment:
Can you format this file a little ie remove extra white space and remove some extra indents?

Reply:
Changes requested has been fixed